test/widgets/actions_test.dart

```diff
@@ -30,19 +41,247 @@ void main() {
 
   Future<void> prepare(WidgetTester tester, {
     UnreadMessagesSnapshot? unreadMsgs,
+    String? ackedPushToken = '123',
```

Comment:
This change can be squashed into the commit after it, which starts using this feature.

(It doesn't touch the same code as the later commit; and it doesn't really need its own commit message, in that it's self-explanatory given the context of that other commit.)